endpoints/endpoints_dispatcher.py

```diff
@@ -82,7 +86,7 @@ def __init__(self, backend_wsgi_app, config_manager=None):
       self._add_dispatcher('%sexplorer/?$' % base_path,
                            self.handle_api_explorer_request)
       self._add_dispatcher('%sstatic/.*$' % base_path,
```

Comment:
I want to avoid sending that stuff to the backend and instead handle it within this file.